proto/src/determined/task/v1/task.proto

```diff
@@ -23,3 +23,13 @@ enum State {
   // The task has begun to exit.
   STATE_TERMINATING = 7;
 }
+
+// A record of Jupyter or TensorBoard.
```

Comment:
A task is a lot more than a record of jupyter or tensorboard, and this API should probably reflect that. I think there is a lot more state we should add (we should at least return everything that is in the DB).